src/components/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import DataHooks from '../../DataHooks';
+import {
+  Card,
+  Cell,
+  dateIndicationProps,
+  FormField,
+  Heading,
+  Input,
+  Layout,
+  Text,
+} from 'wix-style-react';
+import { CommentToString } from '../../utils';
+import { Comment } from '@wix/ambassador-node-workshop-scala-app/rpc';
+import { statusType } from '../StatusMessage/requestStatus';
+
+type CommentsListProps = {
+  comments: Comment[] | undefined;
+  status: statusType;
+};
+
+export default function CommentsList(props: CommentsListProps) {
+  const showNoResultMessage: boolean =
```

Comment:
Here the type is automatically inferred by typescript because of the conditional. So the typing is optional and could be removed to have a cleaner code.
```suggestion
  const showNoResultMessage =
```